[hi] Add content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

BudhirajaMadhav opened the pull request:
This PR adds the localisation for `content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md` in Hindi language. 

Co-Authored by: @AvineshTripathi 

Fixes #30571

budhirajamadhav commented:
/language hi

divya-mohan0209 commented on content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:
```diff
@@ -0,0 +1,72 @@
+
+<!-- overview -->
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं। [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं, जिससे DevOps प्रथाओं का उपयोग करना और विंडोज अनुप्रयोगों के लिए क्लाउड नेटिव पैटर्न का पालन करना आसान हो जाता है। 
```
> [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं

Do we intend for this to be plural?

budhirajamadhav replied:
Can you please clarify, plural for what?

divya-mohan0209 replied:
The word container

divya-mohan0209 commented on content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:
```diff
@@ -0,0 +1,72 @@
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं। [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं, जिससे DevOps प्रथाओं का उपयोग करना और विंडोज अनुप्रयोगों के लिए क्लाउड नेटिव पैटर्न का पालन करना आसान हो जाता है। 
+
+विंडोज-आधारित अनुप्रयोगों और लिनक्स-आधारित अनुप्रयोगों में निवेश वाले संगठनों को अपने कार्यभार को प्रबंधित करने के लिए अलग ऑर्केस्ट्रेटर की तलाश करने की आवश्यकता नहीं है, जिससे ऑपरेटिंग सिस्टम की परवाह किए बिना, उनके परिनियोजन में परिचालन क्षमता में वृद्धि होती है।
```
We either need to Windows or विंडोज across the blog as a standard. Having both will confuse folks.

divya-mohan0209 replied:
Ditto with लिनक्स or Linux.

budhirajamadhav replied:
Yup, I'm aware of this, actually the translator tool does this inconsistency. Will clear once and for all.

divya-mohan0209 commented on content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:
```diff
@@ -0,0 +1,72 @@
+
+<!-- body -->
+
+## कुबेरनेट्स में विंडोज नोड्स
```
We've mentioned Windows before. if there's a standard we're looking to adhere to, we need to conform to one.

divya-mohan0209 commented on content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:
```diff
@@ -0,0 +1,72 @@
+
+## कुबेरनेट्स में विंडोज नोड्स
+
+कुबेरनेट्स में विंडोज कंटेनरों के ऑर्केस्ट्रेशन को सक्षम करने के लिए, अपने मौजूदा लिनक्स क्लस्टर में विंडोज नोड्स शामिल करें। कुबेरनेट्स पर {{< glossary_tooltip text="पॉड्स" term_id="pod" >}} में विंडोज कंटेनरों को शेड्यूल करना लिनक्स-आधारित कंटेनरों को शेड्यूल करने के समान है।
```
> कुबेरनेट्स में विंडोज कंटेनरों के ऑर्केस्ट्रेशन को सक्षम करने के लिए

I am not sure if the plural of containers can be written as कंटेनरों . I believe ऑर्केस्ट्रेशन has a better word too.

budhirajamadhav replied:
> I am not sure if the plural of containers can be written as कंटेनरों .
Couldn't figure out a better alternative.

> I believe ऑर्केस्ट्रेशन has a better word too.
I've used it after confirming in the [slack channel](https://kubernetes.slack.com/archives/CJ14B9BDJ/p1637686690074900). The other suggestion was `आयोजन`, should we go for that?

divya-mohan0209 reviewed (changes requested):
Some initial nits

budhirajamadhav commented:
FIxes #30571

ashish-jaiswar commented on content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:
```diff
@@ -0,0 +1,711 @@
+
+    नोड से स्थानीय नोडपोर्ट पहुंच स्वयं विफल हो जाती है। यह एक ज्ञात है सीमा। NodePort एक्सेस अन्य नोड्स या बाहरी क्लाइंट से काम करता है।
+    
+1. कंटेनर के vNICs और HNS एंडपॉइंट डिलीट जा रहे हैं
```
replace डिलीट by हटाए

BudhirajaMadhav commented on content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:
```diff
@@ -0,0 +1,711 @@
+    
+    यदि आप अभी भी समस्याओं का सामना कर रहे हैं, तो सबसे अधिक संभावना है कि आपका नेटवर्क कॉन्फ़िगरेशन[cni.conf](https://github.com/Microsoft/SDN/blob/master/Kubernetes/flannel/l2bridge/cni/config/cni.conf) कुछ अतिरिक्त ध्यान देने योग्य है। आप इस स्थिर फ़ाइल को कभी भी संपादित कर सकते हैं। NS कॉन्फ़िगरेशन अद्यतन किसी भी नए Kubernetes संसाधनों पर लागू होगा।
+             
+    Kubernetes नेटवर्किंग आवश्यकताओं में से एक (देखें [कुबेरनेट्स मॉडल](/docs/concepts/cluster-administration/networking/)) है क्लस्टर संचार के बिना होने के लिए आंतरिक रूप से एनएटी। इस आवश्यकता का सम्मान करने के लिए, एक है [अपवाद सूची](https://github.com/Microsoft/SDN/blob/master/Kubernetes/flannel/l2bridge/cni/config/cni.conf#L20) उन सभी संचारों के लिए जहां आप नहीं चाहते कि आउटबाउंड NAT हो। तथाप ...
```
That's actually indentation, its likewise in the English version

BudhirajaMadhav commented on content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:
```diff
@@ -0,0 +1,711 @@
+* `terminationGracePeriodSeconds` - यह Windows पर डॉकर में पूरी तरह से लागू नहीं है, [गिटहब मुद्दा] (https://github.com/moby/moby/issues/25982) देखें। आज व्यवहार यह है कि ENTRYPOINT प्रक्रिया CTRL_SHUTDOWN_EVENT भेजी जाती है, तब Windows डिफ़ॉल्ट रूप से 5 सेकंड प्रतीक्षा करता है, और अंत में बंद हो जाता है सामान्य Windows शटडाउन व्यवहार का उपयोग करने वाली सभी प्रक्रियाएं। 5 दूसरा डिफ़ॉल्ट वास्तव मे ...
+* `volumeDevice` - यह एक बीटा सुविधा है, और इसे Windows पर लागू नहीं किया गया है। Windows कच्चे ब्लॉक डिवाइस को पॉड्स में संलग्न नहीं कर सकता है।
+* `volumes`
+  * यदि आप 'emptyDir' वॉल्यूम को परिभाषित करते हैं, तो आप इसके वॉल्यूम स्रोत को 'memory' पर सेट नहीं कर सकते हैं।
```
that is a different point under volumes

BudhirajaMadhav commented:
Hey, @ashish-jaiswar I've fixed the above mentioned nits. Thanks for your review. Also, if you find another sentence that does not make sense(probably happened due to EOL before full stop) then do tell.

jayesh-srivastava reviewed (changes requested):
Hey I noticed that this document contains a mixture of both `Kubernetes` and `कुबेरनेट्स`. We should stick to the hindi one I guess as like the other docs.

anubha-v-ardhan commented:
/hold

BudhirajaMadhav commented:
Hey @anubha-v-ardhan @jayesh-srivastava I was away for some days due to my exams but I've fixed the localisation of `kubernetes` everywhere in the file where it was needed according to my knowledge. Also, I fixed some nits I found in @AvineshTripathi 's part. Let me know if there is something else that needs to be done.

divya-mohan0209 commented:
/remove-lifecycle rotten

divya-mohan0209 reviewed (changes requested):
Kubernetes OR कुबेरनेट्स - please can we standardize across the document?

jayesh-srivastava commented:
> Kubernetes OR कुबेरनेट्स - please can we standardize across the document?

@divya-mohan0209 I think we should use `कुबेरनेट्स` across all docs. Having the word 'Kubernetes' in english wouldn't make sense for hindi content.

divya-mohan0209 commented:
@jayesh-srivastava - I agree. @BudhirajaMadhav - Please can you check and address it? Kindly let us know if you're unable to continue work on this PR.

divya-mohan0209 commented:
Hey @BudhirajaMadhav , please could you comment if you'd be able to continue work on this PR? There has been no activity since January 2022.

divya-mohan0209 commented:
Hello @BudhirajaMadhav, I will be closing the PR since there has been no response from your end & no activity on this PR since Jan. Please reopen when you're comfortable working on the changes.
/close
